### DISCUSSION TITLE
What is the command to upgrade XRDP to the latest version?

### DISCUSSION THREAD

**Discussion by alexlii1971** [neutrinolabs/xrdp#2880] [Q&A]
Hello,

When I try to login from Mac to Ubuntu 22.04 server, I always got the issue like this screenshot:

![Screen Shot 2023-12-18 at 01 30 07](https://github.com/neutrinolabs/xrdp/assets/15896194/3aa28939-bfc4-46a1-b2c6-a839e2f561e6)

![Screen Shot 2023-12-18 at 01 09 59](https://github.com/neutrinolabs/xrdp/assets/15896194/b9efcc06-4540-44a9-addf-10e824dd57e8)

and I just suppose it should be upgraded to the lastest version, right?

If yes, what is the command line to upgrade XRDP to the latest version?

If not, anyway to fix the issue?

Thanks so much

**Comment by matt335672** [comment]
On Ubuntu 22.04, xrdp comes as part of the normal repositories, so will be upgraded when you use the normal apt or apt-get commands to do this.

If the login is failing you may have a keyboard mapping issue. Try putting your password characters in the username field on the login screen to make sure they're coming out as expected.
